be/src/storage/task/engine_clone_task.cpp

```diff
@@ -335,6 +372,13 @@ Status EngineCloneTask::_make_snapshot(const std::string& ip, int port, TTableId
             request.missing_version.push_back(version.first);
         }
     }
+    if (missing_version_ranges != nullptr) {
```

Comment:
if (!missing_version_ranges.empty()) {
    // ...
}

Reply:
`missing_version_ranges != nullptr` is different than `!missing_version_ranges.empty()`

Reply:
oh, i mean the caller also adapt to keep semantic consistency.